components/browser/icons/src/main/java/mozilla/components/browser/icons/utils/DiskCache.kt

```diff
@@ -33,7 +33,7 @@ private const val WEBP_QUALITY = 90
 /**
  * Caching bitmaps and resource URLs on disk.
  */
-internal class DiskCache :
+class DiskCache :
```

Comment:
If we need to open up the visibility here then we should rename this (and `MemoryCache`). For a public API that is way to generic. `IconDiskCache` and `IconMemoryCache` may be enough. In this case the whole interface shenanigans  (`DiskIconLoader.LoaderDiskCache, DiskIconPreparer.PreparerDiskCache, DiskIconProcessor.ProcessorDiskCache`) may be not needed anymore too and only complicate things.